packages/map.vm/map.vm.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>map.vm</id>
-    <version>12.20.21</version>
+    <version>7.26.23</version>
```

Comment:
We should always increase the version, otherwise the old version will be installed (and tested). We need to add this to the linter. We should use the tool version here (and not the date). If there is no tool version, the version should be 0.0.0.DATE, using the 4th segment for the date. This way we ensure we always increase the version.

As the new version will be probably lower after using the correct one, before we can merge this PR, we need to change the version, delete the package in myget and run the CI to ensure this new version is working. We should try to merge this PR fast after deleting the package in myget as it will fail to install during that time.